### ISSUE TITLE
Dark/night mode

### ISSUE THREAD

**Issue by sdcaulley** [jenspfahl/BDT#14]
Please have your app follow device preferences on night mode. It is an accessibility issue. There are some (like myself) that cannot look at something with a black background for long without triggering problems.

**Comment by jenspfahl** [comment]
I added an option in the settings to switch back to Light Mode for those who don't like the dark theme of the app. In v1.2.2.
